shared/js/background/https.es6.js

```diff
@@ -94,9 +91,6 @@ class HTTPS {
                 console.log("HTTPS: updateList() error parsing server response")
             }
         })
-
-        // schedule the next check:
-        setTimeout(this.updateList.bind(this), UPDATE_INTERVAL)
```

Comment:
While we're consolidating stuff, I moved the HTTPS list update to use the extension alarm interface rather than a timeout.